pyproject.toml

```diff
@@ -57,6 +57,8 @@ dev = [
     'pytest-cov',
     'pytest-xdist',
     "pooch>=1.2.0",
+    "nox",
```

Comment:
since `nox` is optional, I don't think it should be included here (just as pre-commit isn't listed here)